resizablecsslayout/src/main/java/com/vaadin/pekka/resizablecsslayout/ResizableCssLayout.java

```diff
@@ -50,9 +50,9 @@ public void onResizeStart(ResizeLocation resizeLocation, int height,
         public void onResizeEnd(int height, int width) {
             pendingHeight = height;
             pendingWidth = width;
-            fireResizeEnd(height, width);
             resizing = false;
             respondResizeAcceptance();
+            fireResizeEnd(height, width);
```

Comment:
Moving this here breaks the cancel-resize feature since the component reports the new size to client in `respondResizeAcceptance()` and the canceling is meant to happen from `ResizeEndEvent` handlers.